app/assets/stylesheets/identity/_vectors.scss

```diff
@@ -0,0 +1,17 @@
+/* fontawesome */
+
+.fa-angle-down {
+  color: $white;
+  font-size: $baseline*2;
+}
+
+/* logo */
+
+.logo {
+  background: url('logo.svg');
```

Comment:
This needs to be written as `background: image-url('logo.svg')` for Rails to pick up the image path correctly

Reply:
Actually, I think it's `background: url(image-path('logo.svg')` for Sass